### DISCUSSION TITLE
Smart Crates besides Smart Playlists

### DISCUSSION THREAD

**Discussion by JONESYdj** [rshea0/enjinn#39] [Ideas]
Hello again,
been testing / using extensively your tool and thank you so much for it. I already have a pretty big enjinn.yaml with many playlists and I'm loving it.

The thing is, I don't need smart playlists since I only use playlists when I want to plan a DJ set or have a pre-planned sequence of songs (and I always do this "by hand"). What I really need is **smart crates**! Having the songs organised in smart crates is a much better workflow when I'm DJing and looking for the best songs to play next.

May I suggest that the software accepts a smartCrates command in the enjinn.yaml, keeping the smartPlaylists as it already does, so that we can have the option?

**Comment by shayded-exe** [comment]
For reasons that I can't disclose, this won't be necessary in the future.

But in the meantime, yeah this would be pretty easy to add. I'll include this in the GUI release. #41

As an aside, how do you like the `enjinn.yaml` format? I've been considering changing it a bit. I opened a new discussion regarding it here https://github.com/rshea0/enjinn/discussions/40

**Comment by shayded-exe** [comment]
This has been implemented for the next release (coming one in the next couple days).

**Reply by JONESYdj** [reply]
Hi. I believe this release never came. I'm still on 1.3.2 and update says it's the latest...

**Reply by shayded-exe** [reply]
Check out the updated documentation. You'll need to uninstall and reinstall the new version now that the architecture is completely different.

Apologies for the late reply. I've been on a bit of a mental health break but I hope to be back to providing updates within a month.
